web/src/context/l10n.jsx

```diff
@@ -108,26 +128,52 @@ function languageToBackend(tag) {
 }
 
 /**
- * This is a helper component to set the language. It uses the
+ * Returns the list of languages from the navigator in RFC 5646 (or BCP 78)
+ * format
+ *
+ * @return {Array<string>} List of languages from the navigator
+ */
+function navigatorLanguages() {
+  return navigator.languages.map(l => l.toLowerCase());
```

Comment:
Important note: the browser preferred languages might not contain the country code, only the language code. For example my personal setting is `['cs', 'en-US', 'en']`. That means Czech, English (US), English (generic, any variant available).

So if the manifest would contain `cs-cz` it would not match. The search for the supported language should be less strict and allow partial match.